guides/manual-installation.md

```diff
@@ -1,19 +1,21 @@
+# Manual Installation of Trento
```

Comment:
Already late, as the file is named as that, but for `Manual` word wouldn't be needed.
This would be `Installing Trento` guide.